website/src/components/edit-word-audio/contributor.tsx

```diff
@@ -323,5 +418,5 @@ async function uploadContributorAudioToS3(user: CognitoUser, data: Blob) {
     })
   )
 
-  return { resourceUrl: `https://d1q0qkah8ttfau.cloudfront.net/${key}` }
+  return { resourceUrl: `https://${process.env["CF_URL"]}/${key}` }
```

Comment:
Does the cloudflare url have `https://` at the front or no? I found some code from @GracefulLemming from four months ago where we started prepending `https://` in a backend process. I'm struggling to find where this secret is managed.